### PR TITLE
[DNM] Testing CI changes

### PR DESCRIPTION
**Pull request by jillr**
Few small meaningless changes to trip test_changed.py in the Zuul job config

##### SUMMARY
test_changed.py

##### ISSUE TYPE
- Bugfix Pull Request

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1071

### CODE REVIEW

**Comment by jillr** [comment]
bot_skip

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

File: tests/integration/targets/ec2_ami/tasks/main.yml [hunk 1]
```diff
@@ -20,6 +20,10 @@
           Name: '{{ ec2_ami_name }}_setup'
       register: setup_vpc
 
+   - name: I am a small change to trigger test_changed.py
+      debug:
```
Comment (tremble) [inline tests/integration/targets/ec2_ami/tasks/main.yml:24]:
```suggestion
     debug:
```

File: tests/integration/targets/ec2_ami/tasks/main.yml [hunk 2]
```diff
@@ -20,6 +20,10 @@
           Name: '{{ ec2_ami_name }}_setup'
       register: setup_vpc
 
+   - name: I am a small change to trigger test_changed.py
+     debug:
+        msg: "Hello"
+
```
Comment (tremble) [inline tests/integration/targets/ec2_ami/tasks/main.yml:26]:
```suggestion
    - name: I am a small change to trigger test_changed.py
      debug:
         msg: "Hello"

```

**Comment by jillr** [comment]
Woohoo, success!
```
2021-08-19 14:54:38.123126 | TASK [ansible-test-splitter : debug]
2021-08-19 14:54:38.225386 | centos-8-stream | ok:
2021-08-19 14:54:38.225668 | centos-8-stream | {
2021-08-19 14:54:38.225742 | centos-8-stream |   "for_zuul_return": {
2021-08-19 14:54:38.225809 | centos-8-stream |     "data": {
2021-08-19 14:54:38.225873 | centos-8-stream |       "child": {
2021-08-19 14:54:38.225935 | centos-8-stream |         "targets_to_test": [
2021-08-19 14:54:38.225998 | centos-8-stream |           [
2021-08-19 14:54:38.226060 | centos-8-stream |             "ec2_ami"
2021-08-19 14:54:38.226121 | centos-8-stream |           ],
2021-08-19 14:54:38.226181 | centos-8-stream |           [
2021-08-19 14:54:38.226241 | centos-8-stream |             "aws_az_info"
2021-08-19 14:54:38.226300 | centos-8-stream |           ],
2021-08-19 14:54:38.226407 | centos-8-stream |           [
2021-08-19 14:54:38.226472 | centos-8-stream |             "ec2_tag"
2021-08-19 14:54:38.226731 | centos-8-stream |           ]
2021-08-19 14:54:38.226934 | centos-8-stream |         ]
2021-08-19 14:54:38.227003 | centos-8-stream |       },
2021-08-19 14:54:38.227063 | centos-8-stream |       "zuul": {
2021-08-19 14:54:38.227123 | centos-8-stream |         "child_jobs": [
2021-08-19 14:54:38.227183 | centos-8-stream |           "ansible-test-cloud-integration-aws-py36_0",
2021-08-19 14:54:38.227242 | centos-8-stream |           "ansible-test-cloud-integration-aws-py36_1",
2021-08-19 14:54:38.227302 | centos-8-stream |           "ansible-test-cloud-integration-aws-py36_2"
2021-08-19 14:54:38.227393 | centos-8-stream |         ]
2021-08-19 14:54:38.227467 | centos-8-stream |       }
2021-08-19 14:54:38.227528 | centos-8-stream |     }
2021-08-19 14:54:38.227588 | centos-8-stream |   }
2021-08-19 14:54:38.227647 | centos-8-stream | }
```

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
Re-using this recent test PR.  `check` pipeline should only test the changed targets, the dependent PR in 1071 should give us an `ondemand` pipeline which can be called to run all targets (effectively the old `ci_complete`).

**Comment by jillr** [comment]
check ondemand

**Comment by jillr** [comment]
It works!  https://ea4ed35f80d76ca798eb-2e92a6d034b069242b31ed6f33f7aea6.ssl.cf2.rackcdn.com/439/00673e62cf5d2bad5411053ecb3aac124f6c4404/ondemand/ansible-test-splitter/4071553/job-output.txt
Will also try on community before merging the zuul change

**Comment by jillr** [comment]
recheck

**Pull request closed without merging**